src/components/ThemeContainer/ThemeContainer.js

```diff
@@ -0,0 +1,38 @@
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+import { getThemeName } from 'selectors/theme';
+import { connect } from 'react-redux';
+
+export class ThemeContainer extends Component {
+  static propTypes = {
+    theme: PropTypes.string.isRequired
+  };
+
+  constructor(props) {
```

Comment:
No need for the constructor if all it does is call super.

Reply:
Removed.